src/resources/js/features/setting/Index.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <v-main>
+    <Block :blockUsers="blockUsers" @release-user-id="getReleaseUserId" />
```

Comment:
`getReleaseUserId `メソッド通さなくても問題なさそうに見えたけど、あった方が良い ❔ 

```suggestion
    <Block :blockUsers="blockUsers" @release-user-id="releaseBlock" />
```